docs_topics/03-lua_versions.md

```diff
@@ -6,4 +6,6 @@ It's tested to work on Debian 9 GNU/Linux with Lua versions:
 * LuaJIT 2.0.0 ... LuaJIT 2.1.0 beta3
 * It may also work on other Lua versions without any guarantees
 
+So basically it should work on any modern Linux-based OS with Lua interpreter available.
```

Comment:
any reason to mention "Linux-based"? works on Mac and Windows as well

Reply:
No reason, my bad. Fixed.